sdk/core/Azure.Core.TestFramework/src/TestEnvironment.cs

```diff
@@ -168,7 +168,11 @@ public virtual TokenCredential Credential
                     _credential = new ClientSecretCredential(
                         GetVariable("TENANT_ID"),
                         GetVariable("CLIENT_ID"),
-                        GetVariable("CLIENT_SECRET")
+                        GetVariable("CLIENT_SECRET"),
+                        new ClientSecretCredentialOptions()
+                        {
+                             AuthorityHost = new Uri(GetVariable("AZURE_AUTHORITY_HOST"))
```

Comment:
Is this always available after New-testResources run?

Reply:
It looks like it is: https://github.com/Azure/azure-sdk-for-net/blob/main/eng/common/TestResources/New-TestResources.ps1#L515